### PR TITLE
Main

### PR DESCRIPTION
**Pull request by GeoffAllix**
UI ipdates and fixes

### CODE REVIEW

File: Labfiles/07-exercise-add-to-index-use-push-api lab-files/OptimizeDataIndexing/obj/Debug/net7.0/OptimizeDataIndexing.AssemblyInfo.cs [hunk 1]
```diff
@@ -13,7 +13,7 @@
 [assembly: System.Reflection.AssemblyCompanyAttribute("OptimizeDataIndexing")]
 [assembly: System.Reflection.AssemblyConfigurationAttribute("Debug")]
 [assembly: System.Reflection.AssemblyFileVersionAttribute("1.0.0.0")]
-[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.0")]
+[assembly: System.Reflection.AssemblyInformationalVersionAttribute("1.0.0+8e767caa68d14a7dcbebb19c12606f640a6efe3d")]
```
Comment (ivorb) [inline Labfiles/07-exercise-add-to-index-use-push-api lab-files/OptimizeDataIndexing/obj/Debug/net7.0/OptimizeDataIndexing.AssemblyInfo.cs:16]: is this going to screw any learners up? I'm not familiar with this notation

File: mslearn-knowledge-mining.generated.sln [hunk 2]
```diff
@@ -0,0 +1,46 @@
+﻿
```
Comment (ivorb) [inline mslearn-knowledge-mining.generated.sln:1]: Generally we don't include sln files in these repos, and let users create their own if desired. we can remove this before merging, or I'll delete it after

File: Labfiles/07-exercise-add-to-index-use-push-api lab-files/OptimizeDataIndexing/obj/Debug/net7.0/OptimizeDataIndexing.AssemblyInfo.cs (hunk 1 above)
Comment (GeoffAllix) [reply Labfiles/07-exercise-add-to-index-use-push-api lab-files/OptimizeDataIndexing/obj/Debug/net7.0/OptimizeDataIndexing.AssemblyInfo.cs]: debug files - now deleted

File: mslearn-knowledge-mining.generated.sln (hunk 2 above)
Comment (GeoffAllix) [reply mslearn-knowledge-mining.generated.sln]: auto-generated. Now deleted

**Pull request merged**